platform/spec/services/temporal/activities/prepare_sync_runs_activity_spec.rb

```
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Temporal::Activities::PrepareSyncRunsActivity do
+  let(:activity) { described_class.new(double("context")) }
```

rubocop (1.68.0)

[convention] Prefer using verifying doubles over normal doubles.

(RSpec/VerifiedDoubles)